app/Jobs/ApplicationDeploymentJob.php

```
@@ -1868,23 +1781,23 @@ private function generate_nixpacks_confs()
                     data_set($parsed, 'phases.setup.aptPkgs', $aptPkgs);
                 }
                 data_set($parsed, 'variables', $merged_envs->toArray());
-
-                if ($this->is_laravel_or_symfony) {
-                    $this->symfony_finetunes($parsed);
+                $is_laravel = data_get($parsed, 'variables.IS_LARAVEL', false);
```

PHPMD (2.15.0)

The variable $is_laravel is not named in camelCase. (undefined)

(CamelCaseVariableName)